completions/python

```diff
@@ -62,15 +62,15 @@ _comp_cmd_python()
             return
             ;;
         !(?(*/)python*([0-9.])|?(*/)pypy*([0-9.])|-?))
-            [[ $cword -lt 2 || ${words[cword - 2]} != -[QWX] ]] && _filedir
+            [[ $cword -lt 2 || ${words[cword - 2]} != -[QWX] ]] && _comp_compgen filedir
```

Comment:
TBH, I'm having a problem understanding what this does.

So if `$prev` is not the python executable or a short option, complete on all files if the word before previous is not `-[QWX]` or we're completing the first arg to python? 

It was added in 9913362b38a01b3a59290f209645028f41b66f65. The commentary in 63ecfc38ba47335e96996e0355925411ecb228f8 provides a hint... that I may not quite get.

But perhaps the intention here is to complete the first arg following a script file with filenames?

I guess a `return` here would be in order.

Reply:
I again looked at it. Maybe it tries to complete the arguments to a Python script (as in `python script.py args[TAB]`). The exclusion of `python` and `pypy` could be to test if `cword` is not the first argument. Anyway, this is just a guess, and the intent of implementing it in this way is still unclear.

The original commit 9913362 also has another missing `return` in the next for-loop, so I think this case branch also missed the `return`.

Fixed in [`bbb0e5f...253609a`](https://github.com/scop/bash-completion/compare/bbb0e5f16a535667a22279ce2fe6e87b2d080c2b..253609ad04003fafcca5e9fcbf44fc8af70ae44a)